WordPress/src/main/java/org/wordpress/android/ui/posts/PostEditorAnalyticsSession.java

```diff
@@ -38,6 +39,7 @@
     private Outcome mOutcome = null;
     private String mTemplate;
     private boolean mHWAccOff = false;
+    private long mStartTime = System.currentTimeMillis();
```

Comment:
I'm a bit worried to have the init here (during `PostEditorAnalyticsSession()` instantiation). It's currently instantiated [here](https://github.com/wordpress-mobile/WordPress-Android/blob/68c042b8daceeac18503d7918add170ed4e3e450/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L530). It could be place earlier, that's not a big issue for what we want to track currently (RN loading times).

Can we add a new function like `startTrackingStartupTime()` that would do this initialization, ideally we should call it in the beginning of `onCreate` but that requires some changes in the constructor.

Reply:
🤔 I agree there's some future risk associated to magically setting this value in initialization, but requiring another call also has the risk of someone refactoring and forgetting to make that call. Although in that case, I guess the error would be more obvious.

I think I'll go with that, but add an assertion so that the session code fails if start gets called and the startup time hasn't been initialized.